Gemfile

```diff
@@ -97,3 +97,4 @@ gem 'pul-assets', github: 'pulibrary/pul_assets'
 gem 'spotlight-resources-iiif', github: 'pulibrary/spotlight-resources-iiif', branch: 'default_vocab'
 gem 'newrelic_rpm'
 gem 'dalli'
+gem 'ruby-prof', require: false
```

Comment:
Yep, I thought about asking for this to be removed, but since it's not required, I think it's fine to leave it in (and it'll make it a little easier to enable profiling going forward).